### PR TITLE
rtt updates/localization for the Crystal Mines

### PR DESCRIPTION
**Pull request by terrapin47**
Standardizes to "Miners' Village"

### CODE REVIEW

File: ETC.tsv [hunk 1]
```diff
@@ -5693,7 +5693,7 @@ ETC_20150317_005692	Body Odor Obtained
 ETC_20150317_005693	Nefenders has fainted!
 ETC_20150317_005694	Wide Area Shooting
 ETC_20150317_005695	Desperate Sprint
-ETC_20150317_005696	Running away for life
+ETC_20150317_005696	Running away in fear
```
Comment (Sunaries) [inline ETC.tsv]: in fear -> to survive/to stay alive

File: ETC.tsv [hunk 2]
```diff
@@ -12473,9 +12473,9 @@ ETC_20150714_012474	Green Demonic Power Crystal
 ETC_20150714_012475	Setting the pots of flame
 ETC_20150714_012476	There are already jewels of prominence nearby
 ETC_20150714_012477	You've discovered the jewel of prominence!
-ETC_20150714_012478	You've defeated the disturbing Beakaras!{nl}Find usable parts from the preliminary purification device
+ETC_20150714_012478	You've defeated the Beakaras!{nl}Get the parts from the spare purifier nearby.
```
Comment (Sunaries) [inline ETC.tsv]: You've defeated the Beakaras! -> You have defeated Bearkaras!

The original does say find for/search, though. You did change it in this line, but not in ETC_20150714_012488.

File: ETC.tsv [hunk 3]
```diff
@@ -12483,7 +12483,7 @@ ETC_20150714_012484	You've found all materials that are needed for the repair!{n
 ETC_20150714_012485	It seems that the demonic power is slowly leaking from the jewels of prominence!{nl}You should find the source and eliminate it fast
 ETC_20150714_012486	Stay in front of the valve to observe
 ETC_20150714_012487	Defeat Beakaras and progress 
-ETC_20150714_012488	You've defeated the Petrified Whale that is disturbing your search for the parts.{nl}Search for the other parts of the main device
+ETC_20150714_012488	You've defeated the Petrified Whale that was impeding your search for the parts.{nl}Search for the other parts of the main device
```
Comment (Sunaries) [inline ETC.tsv]: You've defeated the Petrified Whale -> You have defeated the Stone Whale

File: QUEST.tsv [hunk 4]
```diff
@@ -117,7 +117,7 @@ QUEST_20150317_000116	3. Do not approach closed-off areas. {nl}4. Any suspicious
 QUEST_20150317_000117	Supply Base Soldier
 QUEST_20150317_000118	I've never been in a real fight before so I'm a bit nervous.
 QUEST_20150317_000119	Farm Sentinel
-QUEST_20150317_000120	My friend's in the middle of a battle in Miner's Village. {nl}I've thought of going to help, but the Eastern Woods is in trouble as well, so all I can do is worry.
+QUEST_20150317_000120	My friend's in the middle of a battle at the Miners' Village. {nl}I've thought of going to help, but the Eastern Woods is in trouble as well, so all I can do is worry.
```
Comment (Sunaries) [inline QUEST.tsv]: Original says 동료, which is closer to: colleague, ally, companion. Not friend.

File: QUEST_LV_0100.tsv [hunk 5]
```diff
@@ -4473,7 +4473,7 @@ QUEST_LV_0100_20150317_004472	You finally unleased the first seal. Talk to Gusta
 QUEST_LV_0100_20150317_004473	{memo X}
 QUEST_LV_0100_20150317_004474	{memo X}
 QUEST_LV_0100_20150317_004475	{memo X}
-QUEST_LV_0100_20150317_004476	Go to the Miner's Village Mayor to find out the village's situation.
+QUEST_LV_0100_20150317_004476	Go to the Miners' Village Mayor and find out the village's situation.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: find out about?

**Comment by terrapin47** [comment]
Updated, thanks.

File: ETC.tsv [hunk 6]
```diff
@@ -12473,9 +12473,9 @@ ETC_20150714_012474	Green Demonic Power Crystal
 ETC_20150714_012475	Setting the pots of flame
 ETC_20150714_012476	There are already jewels of prominence nearby
 ETC_20150714_012477	You've discovered the jewel of prominence!
-ETC_20150714_012478	You've defeated the disturbing Beakaras!{nl}Find usable parts from the preliminary purification device
+ETC_20150714_012478	You've defeated the Beakaras!{nl}Search for the parts from the spare purifier nearby.
```
Comment (Sunaries) [inline ETC.tsv]: You've defeated the Beakaras! -> You have defeated Bearkaras!

This message appears centered on your screen IIRC.

File: ETC.tsv [hunk 7]
```diff
@@ -12483,7 +12483,7 @@ ETC_20150714_012484	You've found all materials that are needed for the repair!{n
 ETC_20150714_012485	It seems that the demonic power is slowly leaking from the jewels of prominence!{nl}You should find the source and eliminate it fast
 ETC_20150714_012486	Stay in front of the valve to observe
 ETC_20150714_012487	Defeat Beakaras and progress
```
Comment (Sunaries) [inline ETC.tsv]: Beakaras -> Bearkaras

**Comment by terrapin47** [comment]
:+1:

**Comment by Sunaries** [comment]
Good to merge.

**Comment by terrapin47** [comment]
Resolved conflicts.

**Comment by terrapin47** [comment]
Resolved em.

**Comment by imcgames** [comment]
:+1:

**Pull request merged**